cql/ManageSpecialPopulation.cql

```diff
@@ -737,7 +737,7 @@ define RecommendationForManagingPatientsAfterHysterectomy:
       Rare.AbsenceOrRemovalOfCervix
     ) then 
       {
-        short: 'See Details',
+        short: 'Surveillance',
```

Comment:
Should this include the '3 year' interval as part of the short text to match other Surveillance recommendations?